gdp-src-build/conf/templates/raspberrypi3.local.conf

```diff
@@ -0,0 +1,43 @@
+include templates/local.inc
+
+MACHINE = "raspberrypi3"
+
+GPU_MEM = "128"
+
+CORE_IMAGE_EXTRA_INSTALL += "wayland weston"
+
+MULTI_PROVIDER_WHITELIST += "virtual/libgl virtual/egl virtual/libgles1 virtual/libgles2"
+
+IMAGE_CLASSES = "sdcard_image-rpi-gdp"
+
+KERNEL_DEVICETREE_append = " overlays/vc4-kms-v3d-overlay.dtb"
+
+PREFERRED_VERSION_linux-raspberrypi = "4.4.%"
+PREFERRED_VERSION_mesa = "11.%"
+
+PREFERRED_PROVIDER_virtual/egl = "mesa"
+PREFERRED_PROVIDER_virtual/libgles2 = "mesa"
+PREFERRED_PROVIDER_virtual/libgl = "mesa"
+PREFERRED_PROVIDER_virtual/mesa = "mesa"
+PREFERRED_PROVIDER_jpeg = "jpeg"
+
+#
+# Disk Space Monitoring during the build
+#
+# Monitor the disk space during the build. If there is less that 1GB of space or less
+# than 100K inodes in any key build location (TMPDIR, DL_DIR, SSTATE_DIR), gracefully
+# shutdown the build. If there is less that 100MB or 1K inodes, perform a hard abort
+# of the build. The reason for this is that running completely out of space can corrupt
+# files and damages the build in ways which may not be easily recoverable.
+BB_DISKMON_DIRS = "\
+    STOPTASKS,${TMPDIR},1G,100K \
+    STOPTASKS,${DL_DIR},1G,100K \
+    STOPTASKS,${SSTATE_DIR},1G,100K \
+    ABORT,${TMPDIR},100M,1K \
+    ABORT,${DL_DIR},100M,1K \
+    ABORT,${SSTATE_DIR},100M,1K"
+
```

Comment:
Disk space monitoring? Surely this is not Raspberry Pi specific?

@tom--pollard Please double check everything that can be unified across the platforms and put into the .inc file.

Do we even need this or what is the default if this is not specified in local.conf?
Keep complexity down if possible...